crates/hc_sandbox/tests/cli.rs

```diff
@@ -21,6 +20,8 @@ static HC_BUILT_PATH: Lazy<PathBuf> = Lazy::new(|| {
     let mut manifest_path = PathBuf::from(env!("CARGO_MANIFEST_DIR"));
     manifest_path.push("../hc/Cargo.toml");
 
+    println!("@@ Warning, Building `hc` binary!");
```

Comment:
Warning? Isn't this a good thing?

Reply:
You'd think, right? But, no, building this can take minutes... I thought something was broken, until I started adding debugging like this, and realized I just had to wait it out...